### ISSUE TITLE
PMI DriveAll fails

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/jitutils#237]
Using "PrepAll" works, but using "DriveAll" fails:

```
f:\gh\runtime\artifacts\tests\coreclr\Windows_NT.x64.Debug\Tests\Core_Root>pmi.exe DriveAll System.Text.Encoding.CodePages.dll
ERROR: Unknown command C:\GH\JITUTILS\BIN\PMI.DLL
...
Failed PREPALL on System.Text.Encoding.CodePages.dll
```

**Comment by BruceForstall** [comment]
@AndyAyersMS @dotnet/jit-contrib

**Comment by AndyAyersMS** [comment]
Think the SDK changed at some point to produce exes, so the command line munging between the parent and child PMIs needs adjusting.

**Comment by AndyAyersMS** [comment]
Fixed via #245
